File: two-java-applications/app02/src/main/java/com/example/app02/controller/HelloWorldController.java
package com.example.app02.controller;

import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class HelloWorldController {

    @GetMapping
    public ResponseEntity<String> helloWorld() {
        var a = 1;
        var b = 2;
        return ResponseEntity.ok("Hello World from app02 Java Spring Boot!!!");
    }
}

File: two-java-applications/app03/src/main/kotlin/com/example/app03/controller/HelloWorldController.kt
package com.example.app03.controller

import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/")
class HelloWorldController {
    @GetMapping
    fun getHelloWorld(): ResponseEntity<String> {
        return ResponseEntity.ok("Hello, World from app03 Kotlin Spring Boot")
    }
}
